src/Microsoft.Identity.Client/Platforms/iOS/iOSTokenCacheAccessor.cs

```diff
@@ -215,24 +216,24 @@ public void DeleteAccount(MsalAccountCacheKey cacheKey)
 
             Remove(account, service, type);
         }
-        public ICollection<string> GetAllAccessTokensAsString()
+        public ICollection<MsalAccessTokenCacheItem> GetAllAccessTokens()
         {
-            return GetValues((int)CredentialAttrType.AccessToken);
+            return GetValues((int)CredentialAttrType.AccessToken).Select(x => MsalAccessTokenCacheItem.FromJsonString(x)).ToList();
```

Comment:
nit: I wonder if we really need the ToList() here. Also wonder if we need to return ICollection instead of IEnumerable ... probably not

Reply:
We need the ToList as is since IEnumerable is not an ICollection.  But I agree we don't need ICollection in most/any of these cases.  I'll look at cleaning that up.